blockchain/blockchain.go

```diff
@@ -1047,17 +1047,20 @@ func (b *Blockchain) GetBlockByNumber(blockNumber uint64, full bool) (*types.Blo
 	return b.GetBlockByHash(blockHash, full)
 }
 
-// GetBlocks returns the bunch of block
-func (b *Blockchain) GetBlocks(start, end uint64, full bool) []*types.Block {
+// GetBlocks returns the blocks between start and end block number
+func (b *Blockchain) GetBlocks(start, end uint64, full bool) ([]*types.Block, bool) {
```

Comment:
Why modify the function signature?

The ability to fetch all blocks is directly coupled with the success of this method, so there is no need to return a boolean indicating success - just return `nil`, or maybe an `error` when you fail to fetch all blocks.

cc @Kourin1996

Reply:
Thank you for suggestion, fixed b65283c9249dc562ea3597d3db2fdf69c78213fd